itou/www/siaes_views/forms.py

```diff
@@ -136,26 +138,56 @@ class Meta:
         model = Siae
         fields = [
             "brand",
-            "description",
-            "phone",
-            "email",
-            "website",
             "address_line_1",
             "address_line_2",
             "post_code",
+            "department",  # will not be displayed in the template but filled with clean_department
             "city",
-            "department",
+            "phone",
+            "email",
+            "website",
         ]
+        labels = {
+            "brand": "Nom à afficher",
+            "post_code": "Code postal",
+            "city": "Commune",
+        }
         help_texts = {
-            "brand": (
-                "Si ce champ est renseigné, il sera utilisé en tant que nom "
-                "sur la fiche et dans les résultats de recherche."
-            ),
+            "brand": "Nom présent sur la fiche et dans les résultats de recherche.",
             "description": "Texte de présentation de votre structure.",
-            "phone": "Par exemple 0610203040",
+            "address_line_1": "Appartement, suite, bloc, bâtiment, boite postale, etc.",
+            "address_line_2": "",
             "website": "Votre site web doit commencer par http:// ou https://",
         }
 
+    def clean_department(self):
+        post_code = self.cleaned_data.get("post_code")
+        return department_from_postcode(post_code)
+
+
+class EditSiaeDescriptionForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        for field in self.fields:
+            self.fields[field].required = True
+        self.fields["description"].widget.attrs[
+            "placeholder"
+        ] = "Présentez votre entreprise, votre secteur d’activité, domaines d’intervention, etc."
+        self.fields["provided_support"].widget.attrs[
+            "placeholder"
+        ] = "Indiquer les modalités d’accompagnement, par qui, comment, à quelle fréquence, etc."
```

Comment:
```suggestion
        ] = "Indiquez les modalités d’accompagnement, par qui, comment, à quelle fréquence, etc."
```

Pour rester cohérent avec la ligne 175.

Reply:
OK, ce n'etait pas comme cela dans le mockup mais je modifie.